tests/README.md

```diff
@@ -134,15 +130,18 @@ The project main folders are:
 
 This project uses [loguru](https://github.com/Delgan/loguru) for logging.
 To start logging, just import logger from loguru lib
+
 ```shell
 from loguru import logger
 
 logger.info("Start logging")
 ```
 
-Basic logging configuration is realized through [environment variables](https://github.com/Delgan/loguru/blob/master/loguru/_defaults.py)
+Basic logging configuration is realized
+through [environment variables](https://github.com/Delgan/loguru/blob/master/loguru/_defaults.py)
```

Comment:
revert

Reply:
this is automatically formatted by our pre-commit hook to enforce line length (it won't affect the actually markdown output)